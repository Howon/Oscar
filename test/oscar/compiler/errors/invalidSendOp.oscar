message testNow()


actor Tester() {

  receive = {
    | testNow() => {
        Println("Error");
      }
    | die() => {
      }
  }
}

def main() => unit = {
  pool<Tester> pool1 = spawn pool<Tester>({}, 3);
  message<testNow>() |> pool1;
  message<die>() |>> pool1;
}
