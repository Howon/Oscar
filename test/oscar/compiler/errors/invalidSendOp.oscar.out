Invalid send operation between message<testNow> to pool<Tester>
